cmd/snap-exec/main.go

```diff
@@ -127,10 +127,16 @@ func snapExecApp(snapApp, revision, command string, args []string) error {
 		return fmt.Errorf("cannot find app %q in %q", appName, snapName)
 	}
 
-	cmd, err := findCommand(app, command)
+	cmdAndArgs, err := findCommand(app, command)
 	if err != nil {
 		return err
 	}
+	// strings.Split() is ok here because we validate all app fields
+	// and the whitelist is pretty strict (see
+	// snap/validate.go:appContentWhitelist)
+	cmdArgv := strings.Split(cmdAndArgs, " ")
```

Comment:
In this particular case we do not need to worry about quoting, a quote is not allowed in the field validation we do for commands, so `command: foo "bar baz"` is illegal (same for `'`).